### ISSUE TITLE
Missing file `usage.txt`

### ISSUE THREAD

**Issue by Julio-Guerra** [jsdf/browserify-incremental#17]
I just installed `browserify-incremental@3.0.1`  using `npm install browserify-incremental` and running it throws an error about a missing `usage.txt` file:

````
$./node_modules/browserify-incremental/bin/cmd.js 
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: ENOENT, open 'node_modules/browserify-incremental/bin/usage.txt'
    at Error (native)
````

**Comment by stevenpetryk** [comment]
Why are you running it like that? It's a command line tool.

1. Install it globally: `npm install -g browserify-incremental`
2. Run it anywhere: `browserifyinc [options]`

**Comment by susu** [comment]
Same issue here. I would like to create an npm script into ``package.json``. For example:
```json
...
  "scripts": {
    "build": "browserifyinc --help"
  },
...
```
And then run: ``npm run build``

However it cannot find usage.txt.

**Comment by VictorBjelkholm** [comment]
I have the same issue, not usable at this point.

@stevenpetryk Well, not everyone like to install everything globally. You should be able to install cli's locally as well, because not all projects might require it. I prefer installing dependencies where they are being used instead of globally, and I'm sure I'm not alone.

**Comment by thedufer** [comment]
I agree about not installing globally, but you should be running command-line tools from `node_modules/.bin` rather than reaching into the individual node module directory.  That seems to be working for me.

**Comment by dcousens** [comment]
This can be closed
